recipe/meta.yaml

```diff
@@ -1,42 +1,34 @@
 {% set name = "datafusion" %}
-{% set version = "0.7.0" %}
+{% set version = "23.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 5d85d9328acfb3651eec4dfc0c049de2b9b1bd7d30971f00eac0a901b6148782
+  sha256: b34a819f80fba9a9f3b48e226bf0df53d68c703a2bfabaae7e01e99ca4705c60
 
 
 build:
   number: 0
-  skip: true  # [win]
   script:
     - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
```

Comment:
cc @andygrove is there a reason we want to do this during the conda build versus the sdist/wheel builds? I would think we would want to include this file in the files distributed on PyPI but could be missing something

Reply:
I meant this command that I'm assuming is related to licensing:

```
cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
```

It doesn't look like this is being done when we're building/distributing the source/wheels on PyPI, so I'm wondering if this is something we need specifically for conda-forge or if it would make sense to do this as part of the license generation job of the source/wheel builds:

https://github.com/apache/arrow-datafusion-python/blob/4175134cdf0ed5168282eb88feeb35a7540ac312/.github/workflows/build.yml#L27-L41

I've left things as is so this isn't a blocker for release, mostly wondering if this is a place where we could modify things moving forward